libs/blocks/aside/aside.css

```
@@ -171,7 +171,7 @@
   align-items: center;
 }
 
-.aside:not(.notification) .foreground.container .icon-area {
+.aside:not(.notification):not(.promobar) .foreground.container .icon-area:not(.con-button) {
```

We can group classes to exclude and this also keeps the specificity the same, while having two `:not` pseudo-classes would increase specificity.
```suggestion
.aside:not(.notification, .promobar) .foreground.container .icon-area:not(.con-button) {
```